packages/cascara/src/ui/Pagination/Pagination.js

```diff
@@ -76,7 +78,7 @@ const Pagination = ({ totalRecordCount, state }) => {
 
   return (
     <Boundaries>
-      <div className={styles._}>
+      <Role as={as} className={styles._}>
```

Comment:
We are still muting `className` here

Reply:
This is the latest version:
`<Role {...rest} as={as} className={styles._}>`
Is that right?

Reply:
`<Role {...rest} as={as} className={cx('_', rest.className)}>`
:)